### DISCUSSION TITLE
Path can't create

### DISCUSSION THREAD

**Discussion by tqk2811** [community/community#23789] [Other Feature Feedback, Questions, & Ideas]
<p>I create a path “<a href="http://tqk2811.github.io" rel="nofollow noopener">tqk2811.github.io</a>\.well-known” (has index.html inside) but it not work.<br>
Otherwise \test is worked.</p>
<p>Then is github pages not support this type folder “.well-known”? Thank<br>
<a href="https://github.com/tqk2811/tqk2811.github.io" rel="nofollow noopener">https://github.com/tqk2811/tqk2811.github.io</a></p>

**Comment by Gisgar3** [comment, marked as answer]
<p>Hey, welcome to the <strong>GitHub Support Community</strong>! The page <code>.well-known</code> will not work because GitHub tries to find a file in your repository named <code>.well-known</code>, instead of a directory. However, if you were to remove the period at the beginning of that directory name it should work properly.</p>
<p>Hope this helps!</p>
